content/zh/docs/concepts/policy/limit-range.md

```diff
@@ -0,0 +1,148 @@
+---
+title: Limit Ranges
```

Comment:
@packagewjx you could either translate this title, or adapt it to Chinese by making it singular.

What do you think?

Reply:
Well, I did think of that. It's fine to translate to Chinese. And there are other `limit range`(appears 2 times) in the documentation. I think I should translate those which is separated, and keep those which is not.

By the way, it's Chinese version, "限制范围", is not that intuitive. Maybe there are other better translations?

Reply:
Maybe you can use this: 
```diff
+ title: 限制范围 (Limit Ranges)
```

Reply:
@ydcool It's okay but I haven't seen any other pages use this kind of title. You see, Resource Quota documentation use pure Chinese title.

Here's my opinion. It's okay to use pure Chinese title, because folks don't know what Limit Range is until they read this documentation, even with English version. Same as Resource Quota. So actually it doesn't matter whether it's Chinese title or English title. But at the same time, the title `Limit Ranges` is separated, meaning that it is not a resource name in Kubernetes, LimitRange is. In conclusion, I think pure Chinese is fine.

But I must ask, @tengqm Is it okay to use two languages in title? Am I correct to translate the separated `Limit Ranges` to Chinese instead of keeping it?